apps/src/weblab2/Weblab2View.tsx

```diff
@@ -71,6 +71,7 @@ const defaultConfig: ConfigType = {
 
   labeledGridLayouts,
   activeGridLayout: 'horizontal',
+  showFileBrowser: true,
```

Comment:
should we remove `file-browser` from the grid config for web lab now?

Reply:
d'oh! you're absolutely right. Removed it from the grid and set instructions to go the full height.